packages/editor-ui/src/components/ParameterInputFull.vue

```diff
@@ -18,24 +18,31 @@
 			/>
 		</template>
 		<template>
-			<DraggableTarget type="mapping" :disabled="parameter.noDataExpression || isReadOnly" :sticky="true" :stickyOffset="4" @drop="onDrop">
-				<template v-slot="{ droppable, activeDrop }">
-					<parameter-input
-						ref="param"
-						:parameter="parameter"
-						:value="value"
-						:displayOptions="displayOptions"
-						:path="path"
-						:isReadOnly="isReadOnly"
-						:droppable="droppable"
-						:activeDrop="activeDrop"
-						:forceShowExpression="forceShowExpression"
-						@valueChanged="valueChanged"
-						@focus="onFocus"
-						@blur="onBlur"
-						inputSize="small" />
-				</template>
-			</DraggableTarget>
+				<DraggableTarget
+					type="mapping"
+					:disabled="isDropDisabled"
+					:sticky="!isResourceLocator"
```

Comment:
we should fix this within draggabletarget... so that if it's disabled, it's never sticky.. wonder why this does not happen with other disabled fields though